compiler/crates/extract-graphql/tests/extract/fixtures/tabbed.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+graphql`
+	fragment Test on User {
```

Comment:
This is one of the few times I'm glad Github displays tabs super wide, makes it obvious this is a tab character! 😄